src/NuGet.Core/NuGet.Build.Tasks.Console/MSBuildProjectInstance.cs

```diff
@@ -50,5 +50,17 @@ public IEnumerable<IMSBuildItem> GetItems(string name)
 
         /// <inheritdoc cref="MSBuildItemBase.GetPropertyValue(string)" />
         protected override string GetPropertyValue(string name) => _projectInstance.GetPropertyValue(name);
+
+        public string GetGlobalProperty(string property)
+        {
+            string value = GetPropertyValue(property).Trim();
+
+            if (string.IsNullOrWhiteSpace(value))
+            {
+                return null;
```

Comment:
It looks GetPropertyValue return string.Empty if property is not found. 
So isn't better for keep consistency?

Reply:
Where do you see that?
Can you please link it?

The implementation that I see is: https://github.com/NuGet/NuGet.Client/blob/c3219e49852add8371db25680a300cce5551ef00/src/NuGet.Core/NuGet.Build.Tasks.Console/MSBuildItemBase.cs#L36-L39

When dealing with MSBuild empty is basically non existent.

Reply:
I just looked at [Line29](https://github.com/NuGet/NuGet.Client/blob/c3219e49852add8371db25680a300cce5551ef00/src/NuGet.Core/NuGet.Build.Tasks.Console/MSBuildItemBase.cs#L29) then this one return empty string if property doesn't exist.

![image](https://user-images.githubusercontent.com/8766776/97762568-2ca83500-1ac6-11eb-8a9b-26ce1f16fecb.png)

Reply:
Please **link** the code, not screenshots. 

That's also not our code. 

Our code returns `null` when the value is not set. 

The equivalent method for non-global properties is this one: 

https://github.com/NuGet/NuGet.Client/blob/c3219e49852add8371db25680a300cce5551ef00/src/NuGet.Core/NuGet.Build.Tasks.Console/MSBuildItemBase.cs#L21-L41